quartz.layout.ts

```diff
@@ -41,7 +41,7 @@ export const defaultListPageLayout: PageLayout = {
   left: [
     Component.PageTitle(),
     Component.MobileOnly(Component.Spacer()),
-    Component.Search(),
+    Component.Search({ enablePreview: false }),
```

Comment:
only nit is to make this have no args, we should have preview on by default

```suggestion
    Component.Search(),
```

Reply:
oh right. forgot to remove this. 😄